.eslintrc.js

```diff
@@ -21,15 +21,12 @@ module.exports = {
 	plugins: ["react", "jest", "jsx-a11y", "react-hooks", "sort-exports"],
 	rules: {
 		"global-require": "off",
-		indent: ["error", "tab"],
```

Comment:
todo: update this for the blocks as well in a separate pr

Reply:
no issue in the blocks around this

Reply:
howardj@arc-20750 fusion-theme-blocks % npx eslint-config-prettier .eslintrc.js
No rules that are unnecessary or conflict with Prettier were found.